packages/loading-indicator/README.md

```diff
@@ -0,0 +1,46 @@
+# Loading Indicator
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/loading-indicator.svg)
+
+#### [View on MongoDB.design](https://www.mongodb.design/component/loading-indicator/example/)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/loading-indicator
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/loading-indicator
+```
+
+## Spinner
+
+```
+<Spinner
+  variant="large"
+  description=”Loading…”
+/>
+```
+
+| Prop           | Type      | Description                                                                        | Default   |
+| -------------- | --------- | ---------------------------------------------------------------------------------- | --------- |
+| `variant`      | `Variant` | Determines the size or orientation of the spinner and description text             | `default` |
+| `description`  | `string`  | Description text                                                                   | `-`       |
+| `sizeOverride` | `number`  | An override for the spinner animation’s size in pixels. Intended for internal use. | `-`       |
```

Comment:
Don't document this if it's for internal use only

Reply:
I feel like it should still be documented for the sake of completion?

Reply:
I think if we document it then people will use it